cmake/BuildLibXXhash.cmake

```diff
@@ -0,0 +1,23 @@
+macro(libxxhash_build)
+    set(xxhash_src third_party/xxHash/xxhash.c)
+
+    if (CC_HAS_WNO_IMPLICIT_FALLTHROUGH)
+        set_source_files_properties(${xxhash_src}
+                PROPERTIES COMPILE_FLAGS
+                -Wno-implicit-fallthrough)
+    endif()
+
+    # Specify "XXH_NAMESPACE" to prevent symbol clash with zstd
+    # that also uses xxhash.
+    # Remaining properties are the same as for zstd
+    # (see cmake/BuildZSTD.cmake).
+    set_source_files_properties(${xxhash_src}
+            PROPERTIES COMPILE_FLAGS "-Ofast -DXXH_NAMESPACE=tnt_")
+
+    add_library(xxhash STATIC ${xxhash_src})
+    set(XXHASH_LIBRARIES xxhash)
+    set(XXHASH_INCLUDE_DIRS
+            ${CMAKE_CURRENT_SOURCE_DIR}/third_party/xxHash)
+    include_directories(${XXHASH_INCLUDE_DIRS})
+    add_dependencies(build_bundled_libs xxhash)
```

Comment:
It seems, this line is duplicated with the same one in ${PROJECT_ROOT}/CMakeLists.txt.

Reply:
Oh, seems zstd contains the same line. Removed. Thanks.

Reply:
Do you think such a patch is needed https://github.com/tarantool/tarantool/commit/72a850d2f4b26df5fcc9a46ff86703b29226e52c?

Reply:
Seems so.

Reply:
Good. I opened separate PR for it https://github.com/tarantool/tarantool/pull/6161